discord/ui/item.py

```diff
@@ -89,6 +90,9 @@ def is_dispatchable(self) -> bool:
     def is_persistent(self) -> bool:
         return self._provided_custom_id
```

Comment:
If it's not dispatchable, then it's persistent :)
(i was having issues with CV2 not being allowed to be persistent, this was my fix)
```suggestion
        return not self.is_dispatchable() or self._provided_custom_id
```